recommendation/system.web.json

```diff
@@ -76,14 +83,66 @@
               "Type": "Method",
               "Value": "HtmlEncoder.Default.Encode",
               "Description": "Replace HtmlEncode with a new method: HtmlEncoder.Default.Encode.",
-              "ActionValidation":{"Contains":"HtmlEncoder.Default.Encode", "NotContains":""}
+              "ActionValidation": {
+                "Contains": "HtmlEncoder.Default.Encode",
+                "NotContains": ""
+              }
             },
             {
               "Name": "AddDirective",
               "Type": "Using",
               "Value": "System.Text.Encodings.Web",
               "Description": "Add System.Text.Encodings.Web.",
-              "ActionValidation":{"Contains":"using System.Text.Encodings.Web;", "NotContains":""}
+              "ActionValidation": {
+                "Contains": "using System.Text.Encodings.Web;",
+                "NotContains": ""
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Type": "Method",
+      "Name": "System.Web.HttpContext.Response.Write(string)",
+      "Value": "System.Web.HttpContext.Response.Write(string)",
+      "KeyType": "Name",
+      "ContainingType": "IHttpHandler",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Replace HttpContext.Response.Write with a new method HttpContext.Response.WriteAsync in namepsace Microsoft.AspNetCore.Http.",
+          "Actions": [
+            {
+              "Name": "ReplaceMethodWithObject",
+              "Type": "Method",
+              "Value": "Microsoft.AspNetCore.Http.HttpContext.Response.WriteAsync",
```

Comment:
You probably only want to replace the method only here rather than the whole object otherwise you will end up with the namespace in the final code rather than the variable standing in for httpcontext. And since you are adding the new directive to ensure the new httpcontext comes from aspnetcore you should be ok.